### PR TITLE
Create public_ld_datasets.rst

### PR DESCRIPTION
**Pull request by pouyak**

### CODE REVIEW

**Comment by deflaux** [comment]
Thanks @pouyak !

**Comment by pgrosu** [comment]
Nicole, please wait there is an issue with this!

File: docs/source/use_cases/linkage_disequilibrium/public_ld_datasets.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+
+Linkage disequilibrium is run separately for each `super population and sub population <http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/integrated_call_samples_v3.20130502.ALL.panel>`_ using the `Linkage Disequilibrium pipeline <compute_linkage_disequilibrium.rst>`_. LD is computed for all pairs of variants within a window of 1,000,000 bp (1 megabase) and all pairs with absolute allelic correation of 0.4 are retained. 
+
+The `output files <https://pantheon.corp.google.com/storage/browser/genomics-public-data/linkage-disequilibrium/1000-genomes-phase-3/ldCutoff0.4_window1MB/>`_ are split by chromosome with `output columns <https://github.com/googlegenomics/linkage-disequilibrium#linkage-disequilibrium-calculation-pipeline>`_ indicating the identity of each pair of values and the resulting LD value. The output files have also been `loaded into BigQuery <https://bigquery.cloud.google.com/dataset/genomics-public-data:linkage_disequilibrium_1000G_phase_3?pli=1>`_ with the same columns. Examples of using BigQuery to analyze LD are `available as datalabs <https://github.com/googlegenomics/linkage-disequilibrium/tree/master/datalab>`_.
```
Comment (pgrosu) [inline docs/source/use_cases/linkage_disequilibrium/public_ld_datasets.rst]: @pouyak You'll need to point this link to somewhere publicly, as now it's pointing internally to MOMA:

https://pantheon.corp.google.com/storage/browser/genomics-public-data/linkage-disequilibrium/1000-genomes-phase-3/ldCutoff0.4_window1MB/

moma

Paul

Comment (pgrosu) [inline docs/source/use_cases/linkage_disequilibrium/public_ld_datasets.rst]: I take it that after the staging process, the following two links will work...

https://github.com/googlegenomics/linkage-disequilibrium#linkage-disequilibrium-calculation-pipeline

https://github.com/googlegenomics/linkage-disequilibrium/tree/master/datalab

**Comment by deflaux** [comment]
@pgrosu thanks! all will be well soon.  As you've noticed, we are actively working on this.  The staging site is a good place to integrate and do final checks on new material..

**Comment by pgrosu** [comment]
Sounds good Nicole :)

Thanks,
~p

**Pull request merged**